src/containers/SubjectPage/SubjectPage.jsx

```diff
@@ -38,6 +38,14 @@ const SubjectPage = ({
     },
   });
 
+  useEffect(() => {
+    if (data?.subject && !getFiltersFromUrl(location)) {
+      history.replace({
+        search: `?filters=${data.subject.filters[0].id}`,
```

Comment:
Er vi sikre på at subjet har filters?
```suggestion
        search: `?filters=${data.subject.filters?.[0]?.id}`,
```

Reply:
Er vi ikke? Da bør vi sikkert heller droppe å legge på parameter i det tilfellet